wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/ScriptEngineInterlayerManager.java

```diff
@@ -0,0 +1,31 @@
+package ru.ydn.wicket.wicketconsole;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ScriptEngineInterlayerManager {
+
+	public static final ScriptEngineInterlayerManager INSTANCE = new ScriptEngineInterlayerManager();
+
+	Map<String,IScriptEngineInterlayer> interlayers;
+	
+	private ScriptEngineInterlayerManager() {
+		interlayers = new HashMap<String,IScriptEngineInterlayer>();
```

Comment:
If I understood this correctly: execution context will be the same for all users who is working in parallel. And that's definetly wrong.